docs/source/gcp/gettingstarted.rst

```diff
@@ -12,9 +12,10 @@ writing policies which interact with GCP related resources.
 Install GCP Plugin
 ------------------
 
-First, ensure you have :ref:`installed the base Cloud Custodian application <install-cc>`. 
-Cloud Custodian is a Python application that supports Python 2 and 3 on Linux and Windows. 
-We recommend using Python 3.6 or higher.
+First, ensure you have :ref:`installed the base Cloud Custodian application
+<install-cc>`. Cloud Custodian is a Python application and must run on an
+`actively supported <https://devguide.python.org/#status-of-python-branches>`_
```

Comment:
nit.. i'm a little ambigious on the link target being elided to actively supported versus typed out here as actively supported is context dependent (ala linux distros cough rhel supporting python 2.7 in rhel 7 to 2024 :cry: ). but :man_shrugging: seems okay though.

Reply:
OK added a snippet, I could go either way.